README.md

```diff
@@ -198,11 +198,13 @@ options:
 
   # Automatically request reviewers when a Pull Request is opened
   # if this rule is pending, there are no assigned reviewers, and if the
-  # Pull Request is not in Draft. Reviewers are selected randomly
-  # based on the set of requirements for this rule.
+  # Pull Request is not in Draft. If add_everyone is false,
+  # Reviewers are selected randomly based on the set of requirements for this rule.
+  # If add_everyone is true, all possible reviewers are requested for review.
   request_review:
     # False by default
     enabled: true
+    add_everyone: false
```

Comment:
I'm not sold on the name of this, but I'm not sure what else to call it.

Reply:
Perhaps it could be `everyone` or `all`. I think it should also default to true instead of `false` to me it feels like it the most expected behaviour if I was turning this feature on

Reply:
I like the idea of an enum here!

Reply:
As a bystander, I can't imagine a case when you'd want the current behavior of picking a random user. The point of being an owner is to know when code changes

Reply:
@akroy being an owner of a repo in GitHub brings the ability to do admin things to the repo. There are times when the repo admin (i.e. owner) is there to ensure settings are enforced but they don't actually participate in the code being generated. An example of this might be that there are no human owners on the repo as the admin duties are delegated to bots which maintain the settings. This is certainly the case on some of the repos I work on.

Reply:
Sorry to clarify, I'm talking about being a code owner, not a repo owner. (the native GitHub feature that we're replacing with Policy-Bot) I think the goal there is see the code changes. If you didn't want to see the code changes, and the work was done by bots, would you still be a code owner/policy member?

At the very least, it seems like the default repo's code owners want to see changes to the code they own?